Source/WebKit/Resources/SandboxProfiles/ios/com.apple.WebKit.WebContent.sb.in

```diff
@@ -716,16 +716,15 @@
     (sysctl-name-prefix "hw.perflevel") ;; <rdar://problem/76782530>
 )
 
-(deny sysctl-read (with no-report)
-    (sysctl-name "vm.task_no_footprint_for_debug"))
```

Comment:
Yes, this will now be covered by the generic deny case, so we don't need a specific rule.

Reply:
Yes, this will now be covered by the generic deny case, so we don't need a specific rule.

> Looks good -- thank you for answering my questions! :-)

NP, thanks for the approval!